### ISSUE TITLE
I want to modify the request header

### ISSUE THREAD

**Issue by tiansin** [apifytech/proxy-chain#24]
I need to modify the request header in the `prepareRequestFunction` callback. Can I do this?

**Comment by jancurn** [comment]
Yes, if you modify the request headers (request.rawHeaders), they will be
passed to the target host. But please note that this will only work for
HTTP requests, not HTTPS which cannot be intercepted.
